app/assets/javascripts/admin/addon/controllers/admin-whats-new.js

```diff
@@ -0,0 +1,16 @@
+import Controller from "@ember/controller";
+import { action } from "@ember/object";
+
+export default class AdminWhatsNewController extends Controller {
+  @action
+  checkForUpdates() {
+    if (this.checkFeaturesCallback) {
+      this.checkFeaturesCallback({ forceRefresh: true });
+    }
+  }
+
+  @action
+  bindCheckFeatures(checkFeaturesCallback) {
+    this.checkFeaturesCallback = checkFeaturesCallback;
+  }
```

Comment:
I'm not sold on this "pass an action between sibling components via the parent controller" pattern 😅

Thinking of alternatives: a dedicated service is too verbose imo; maybe an app-event?
(@davidtaylorhq, thoughts?)

Reply:
Yeah I'm not sure I like it much either this just felt like the easiest option, happy to hear other suggestions.